.github/workflows/ci.yml

```diff
@@ -0,0 +1,155 @@
+           updated_charts=$(echo "$updated_charts" | sed "s/-helm//g")
+
+           if [[ "$updated_charts" == "[]" ]]; then 
+            echo ::set-output name=no_changes::"true"
+           fi
+           echo ::set-output name=matrix::{\"chart\":$(echo $updated_charts)}\"
+           
+           echo "Charts array to run CI on: $updated_charts"
+  
+  check:
+    name: Check Helm Chart '${{ matrix.chart }}'
+    if: ${{ needs.detect-repo-changes.outputs.no-changes != 'true' }}
+    needs: [dependencies, detect-repo-changes]
+    runs-on: ubuntu-20.04
+    env:
+      HELM_CHART: ${{ matrix.chart }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Installing kubeval
+        run: helm plugin install https://github.com/instrumenta/helm-kubeval
+      - name: Lint helm chart 
+        run: make helm-lint
+      - name: Validate helm chart 
+        run: make helm-validate
+    strategy:
+      matrix: ${{ fromJson(needs.detect-repo-changes.outputs.helms-changed) }}
+
+ 
+  # test-upgrade:
+  #   name: Upgrade Helm Chart '${{ matrix.chart }}'
+  #   if: ${{ needs.detect-repo-changes.outputs.no-changes != 'true' }}
+  #   needs: [check, detect-repo-changes, dependencies]
+  #   runs-on: ubuntu-20.04
+  #   env:
+  #     HELM_CHART: ${{ matrix.chart }}
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v2
+  #     - name: Install k3d 
+  #       run: |
+  #         export PATH=".bin:$PATH"
+  #         curl -s https://raw.githubusercontent.com/k3d-io/k3d/v5.4.4/install.sh | USE_SUDO="false" K3D_INSTALL_DIR=".bin" bash
+  #     - name: Test upgrade for helm chart 
+  #       run: make helm-upgrade
+  #   strategy:
+  #     matrix: ${{ fromJson(needs.detect-repo-changes.outputs.helms-changed) }}
+ 
+  test-install:
+    name: Install Helm Chart '${{ matrix.chart }}'
+    if: ${{ needs.detect-repo-changes.outputs.no-changes != 'true' }}
+    needs: [check, detect-repo-changes, dependencies]
+    runs-on: ubuntu-20.04
+    env:
+      HELM_CHART: ${{ matrix.chart }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install k3d 
+        run: |
+          export PATH=".bin:$PATH"
+          curl -s https://raw.githubusercontent.com/k3d-io/k3d/v5.4.4/install.sh | USE_SUDO="false" K3D_INSTALL_DIR=".bin" bash
+      - name: Test install for helm chart 
+        run: make helm-test
+    strategy:
+      matrix: ${{ fromJson(needs.detect-repo-changes.outputs.helms-changed) }}
+
+  release:
+    name: Release
+    if: ${{ github.ref_type == 'tag' }}
+    # needs: [test-install, test-upgrade]
+    needs: [test-install]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Define current tag version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: Generate documentation 
```

Comment:
We'd like docs to be generated not on release, but on post push to the master branch ;)